mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -159,6 +163,8 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "active_swaps" => handle_mmrpc(ctx, request, active_swaps_rpc).await,
         "add_delegation" => handle_mmrpc(ctx, request, add_delegation).await,
         "add_node_to_version_stat" => handle_mmrpc(ctx, request, add_node_to_version_stat).await,
+        "approve" => handle_mmrpc(ctx, request, approve_rpc).await,
+        "allowance" => handle_mmrpc(ctx, request, allowance_rpc).await,
```

Comment:
Should have more descriptive name, maybe `approve_token` and `get_token_allowance` or something similar.

Reply:
evm_approve, evm_allowance maybe?

Reply:
I think `approve_token` and `get_token_allowance` is good enough since we can add more token types in the future like SPL ref. https://github.com/KomodoPlatform/komodo-defi-framework/pull/2222#discussion_r1833025046

Reply:
fixed